src/SlackConnector/Models/SlackAttachmentActionStyle.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace SlackConnector.Models
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum SlackAttachmentActionStyle
```

Comment:
I have followed the implicit convention of naming the enum by using the 'parent' class name suffixed with property name, as on SlackChatHubType (used on SlackChatHub class' Type property).